### PR TITLE
Add quiz taskable

### PR DESCRIPTION
**Pull request by ohenrik**
This PR also includes the PR #69 Answers can now be updated

### CODE REVIEW

File: spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb [hunk 1]
```diff
@@ -56,5 +57,49 @@ module Tasuku
...
+
+        before { request.env['HTTP_REFERER'] = 'http://example.org' }
+        before { allow(subject).to receive(:current_user).and_return(user) }
+
+        #it_behaves_like 'redirectable' do
+        #  let(:action) { :update }
+        #  let(:verb)   { :put }
+        #  let(:update_answer) { true }
+        #end
+        # TODO: Need to solve this later
```
Comment (jgorset) [inline spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb]: What's the issue with this?

File: spec/factories/tasuku_taskables_questions.rb [hunk 2]
```diff
@@ -5,13 +5,14 @@
     text "MyText"
     multiple true
 
-    factory :tasuku_taskables_question_with_options, aliases: [:question_with_options] do
+    factory :tasuku_taskables_question_with_options, aliases: [:question_with_options] do
```
Comment (jgorset) [inline spec/factories/tasuku_taskables_questions.rb]: Now I'm just being a dick, but there's whitespace at the end of this line that shouldn't be there.

File: spec/factories/tasuku_taskables_quizzes.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  factory :tasuku_taskables_quiz, aliases: [:quiz], :class => 'Tasuku::Taskables::Quiz' do
+    text "Quiz text"
+    description "Quiz description"
+
+    factory :tasuku_taskables_quiz_with_questions, aliases: [:quiz_with_questions] do
+      ignore do
+        questions_count 3
+        options_count 3
+        correct_option 1
+      end
```
Comment (jgorset) [inline spec/factories/tasuku_taskables_quizzes.rb]: I love this kind of thing.

File: lib/tasuku.rb [hunk 4]
```diff
@@ -12,9 +12,20 @@ def configure
     end
 
     def config
-      @config ||= Configatron::Store.new author: :current_user
-      @config
```
Comment (jgorset) [inline lib/tasuku.rb]: It's nice to drop this line.

File: README.md [hunk 5]
```diff
@@ -80,6 +80,8 @@ helpers with `main_app` so Ruby on Rails knows where to look.
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
 
+Note: Remember to create the test database under `spec/dummy`, using `bundle exec rake db:setup RAILS_ENV=test`  
+
```
Comment (jgorset) [inline README.md]: :+1:

**Comment by jgorset** [comment]
I like this pull request. This is a good pull request.

**Comment by mathiasbno** [comment]
:camel: :sagittarius:

File: spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb (hunk 1 above)
Comment (ohenrik) [inline spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb]: I was not able at the time to add it conditional tests to this part so i commented it out for questions that can be updated. We might need to create a new it_behaves_like to test this behavior. maybe 'redirectable_update_answer'

File: spec/factories/tasuku_taskables_questions.rb (hunk 2 above)
Comment (ohenrik) [inline spec/factories/tasuku_taskables_questions.rb]: Haha, ok, noted :p I'll fix it!

File: spec/factories/tasuku_taskables_quizzes.rb (hunk 3 above)
Comment (ohenrik) [inline spec/factories/tasuku_taskables_quizzes.rb]: :dancer:

**Comment by JozoSalt** [comment]
Is it so nice, that no one wants to merge it ? :)

**Comment by jgorset** [comment]
Huhhhh – yeah, why did we never merge this?

**Comment by sindrenm** [comment]
LOL.

**Comment by toothfairy** [comment]
EL OL

**Comment by toothfairy** [comment]
@JozoSalt now it's even more awesome

**Comment by ohenrik** [comment]
:cake: :tada:

**Comment by JozoSalt** [comment]
@toothfairy :+1:

**Pull request merged**